calabash/Classes/FranklyServer/Routes/LPConditionRoute.m

```diff
@@ -90,9 +91,15 @@ - (void) checkConditionWithTimer:(NSTimer *) aTimer {
       for (id v in result) {
         if ([v isKindOfClass:[UIView class]]) {
           UIView *view = (UIView *) v;
-          if ([[view.layer animationKeys] count] > 0) {
-            self.stablePeriodCount = 0;
-            return;
+          NSArray *animationKeys = [[view.layer animationKeys] copy];
```

Comment:
@JoeSzymanski Memory leak. :(

I think I can convert this file to ARC.

This was a fantastic addition.  Thanks.